### PR TITLE
Support glob pattern to exclude unknown validation

### PR DESCRIPTION
**Pull request by angelozerr**
Support glob pattern to exclude unknown validation

Signed-off-by: azerr <azerr@redhat.com>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -235,7 +235,7 @@
           "items": {
             "type": "string"
           },
-          "description": "Array of properties to ignore for unknown Quarkus properties validation.",
+          "description": "Array of properties to ignore for unknown Quarkus properties validation. Pattern can be used ('*' = any string, '?' = any character)",
```
Comment (fbricon) [inline package.json]: Patterns can be used

File: README.md [hunk 2]
```diff
@@ -60,7 +60,7 @@ Default is `warning`.
 * `quarkus.tools.validation.syntax.severity` : Validation severity for Quarkus property syntax checking.
 Default is `error`.
 * `quarkus.tools.validation.unknown.severity` : Validation severity for unknown Quarkus properties. Default is `warning`.
-* `quarkus.tools.validation.unknown.excluded` : Array of properties to ignore for unknown Quarkus properties validation. Default is `[]`.
+* `quarkus.tools.validation.unknown.excluded` : Array of properties to ignore for unknown Quarkus properties validation. Pattern can be used ('*' = any string, '?' = any character). Default is `[]`.
```
Comment (fbricon) [inline README.md]: Patterns can be used

Comment (angelozerr) [reply README.md]: @fbricon fixed

File: package.json (hunk 1 above)
Comment (angelozerr) [reply package.json]: @fbricon fixed

**Pull request merged**